plugins/outputs/riemann/riemann.go

```diff
@@ -145,14 +145,14 @@ func (r *Riemann) buildRiemannEvents(m telegraf.Metric) []*raidman.Event {
 			Tags:       r.tags(m.Tags()),
 		}
 
-		switch value.(type) {
+		switch value := value.(type) {
```

Comment:
Please avoid overriding the variable. Use 
```suggestion
		switch v := value.(type) {
```
or similar instead.